R/combinedscatterplot.R

```diff
@@ -854,18 +856,23 @@ fitLines <- function(scatter.colors, scatter.colors.as.categorical, scatter.grou
     fit.lower.bound <- vector("list", num.lines)
     fit.upper.bound <- vector("list", num.lines)
     fit.line.names <- character(num.lines)
+    fit.line.colors <- character(num.lines)
     fit.ci.fill.colors <- character(num.lines)
     fit.ci.label.colors <- character(num.lines)
 
-    if (is.null(fit.line.colors))
-        fit.line.colors <- colors
+    if (is.null(input.fit.line.colors))
+        input.fit.line.colors <- colors
```

Comment:
Where does input.fit.line.colors come from?

Reply:
I renamed the input parameter for this function to distinguish it from the output `fit.line.color` (they have different lengths and are not the same).